db/mongodb.go

```diff
@@ -0,0 +1,92 @@
+package mongodb
+
+import (
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
+const (
+	cronCollection = "cronjobs"
+)
+
+type DB struct {
+	DB      *mgo.Database
+	Session *mgo.Session
+}
+
+type CronJob struct {
+	ID       bson.ObjectId `bson:"_id,omitempty"`
+	IsActive bool          `bson:"active"`
+	Function string        `bson:"function"`
+	Workload interface{}   `bson:"workload"`
+	CronTime string        `bson:"time"`
+	TimeZone string        `bson:"tz"`
+	Created  time.Time     `bson:"created"`
+}
+
+func New(mongoURL string, dbName string) (*DB, error) {
+	session, dialErr := mgo.Dial(mongoURL)
+	if dialErr != nil {
+		return nil, dialErr
+	}
+	cleverDb := session.DB(dbName)
+	return &DB{DB: cleverDb, Session: session}, nil
+}
+
+func (db *DB) GetDistinctActiveJobs() ([]string, error) {
```

Comment:
the methods here should use a clone of the root session, e.g. https://github.com/Clever/hall-monitor/blob/36acf8691921c0a69d86e35d455ce7bcc9d8471d/vendor/github.com/Clever/district-user-service/userstore/districtadmin.go#L209-L210

Otherwise all of the methods will share the same socket (see https://godoc.org/gopkg.in/mgo.v2#Session)